### PR TITLE
Remove connection factory when crc has instance

### LINKED ISSUE #122: Odd behaviour with 'Create new' after an instance exists
If I have a running OpenShift Local instance, the `Create new` button is still enabled, and leads to a screen where I can stop the running instance but nothing else. Supporting only one instance at a time is fine, but then the button should probably be disabled or removed after creating the first one.

I'm aware this could mean there's a feature requirement on Podman Desktop, but opening here since this is where the issue surfaces and would need to make a change either way.

### PR DESCRIPTION
**Pull request by evidolob**
Fix #122 

Demo:

https://github.com/crc-org/crc-extension/assets/929743/c65a740e-7f04-4df7-a5e3-6c27d693e312

### CODE REVIEW

**Comment by gbraad** [comment]
The last step is confusing; the `close` and `create` buttons have been removed and the user has to resort to using the `cross` (X) to close the creation pane.

**Comment by evidolob** [comment]
That is out of our control, that is PD does.
I don't change that part in this PR. This PR is only about hiding `Create new...` button, when CRC has an instance.

File: yarn.lock [hunk 1]
```diff
@@ -14,6 +14,11 @@
   dependencies:
     "7zip-bin" "5.1.1"
 
+"@aashutoshrathi/word-wrap@^1.2.3":
```
Comment (gbraad) [inline yarn.lock:17]: ? where is this used?

Comment (gbraad) [reply yarn.lock:17]: something as a dependency from PD?

**Review by gbraad — APPROVED** [review]

**Pull request merged**